src/main.cpp

```diff
@@ -27,6 +27,38 @@ void initialize_pins()
 	}
 }
 
+float calculate_frequency(double velocity)
+{
+	// Take the equation the propulsion subteam provided, substitute the
+	// constants, and simplify. The resulting equation is a quadratic, where
+	// x is the frequency and the velocity is a constant. This means that
+	// we can use the quadratic formula and solve for the appropriate frequency.
+	// Note that since this is a quadratic, there are two solutions. We
+	// default to the solution that has the higher frequency for increased
+	// stability.
+
+	// Run the quadratic formula on these coefficents to solve for
+	// u = ((L*x)/(2*pi)) - v_r and solve for x.
+
+	int N = 183;			  // number of turns per phase
+	int I = 54;				  // RMS current through coil (amps)
+	float D = 0.1;			  // stator thickness (meters)
+	float d_r = 0.01048;	  // track thickness (meters)
+	float L = 0.55;			  // stator length (meters)
+	float sigma = 3.03e7;	  // track conductance/meter (Siemens/meter)
+	float g = 0.02548;		  // air gap between stators (meters)
+	float mu = 1.25663753e-6; // permeability of air (Henries per meter)
+	int F_thrust = 1200;	  // constant thrust (Newtons)
```

Comment:
Note for later: the target thrust value is probably not correct, given the constants above, should be closer to 2700.